multi_import/fields.py

```diff
@@ -0,0 +1,129 @@
+from rest_framework import fields  # noqa
+
+from multi_import.utils import normalize_string
+
+__all__ = [
+    'empty',
+    'FieldMixin',
+    'Field',
+    'BooleanField',
+    'CharField',
+    'ChoiceField',
+    'DateField',
+    'DateTimeField',
+    'DecimalField',
+    'DurationField',
+    'EmailField',
+    'FileField',
+    'FilePathField',
+    'FloatField',
+    'ImageField',
+    'IntegerField',
+    'IPAddressField',
+    'ModelField',
+    'NullBooleanField',
+    'SlugField',
+    'TimeField',
+    'URLField',
+]
+
+
+empty = fields.empty
+
+
+class FieldMixin(object):
```

Comment:
I don't agree with the approach of this mixin. We shouldn't need this. The consumer of the field can use it as a string. From what you described to me; for things like tags, we should be using specialized fields e.g. StringTagField.